### ISSUE TITLE
No parse result when using single brackets in a string used in a filter

### ISSUE THREAD

**Issue by intersel** [modxcms/revolution#16391]
### Summary
When a string contains brackets like '[hello]' and is used in a filter like "empty", there is no result or wrong result.

### Step to reproduce
Create a resource with a void template

Put the following text in the content:

```
longtitle: [[*longtitle]]<br>
Result1:[[*longtitle:empty=`title`]]<br>
Result2:[[*longtitle:empty=`[title]`]]<br>
```

### Observed behavior
```
longtitle: <br>
Result1:title<br>
Result2:<br>
```

### Expected behavior
```
longtitle: <br>
Result1:title<br>
Result2:[title]<br>
```

### Environment
MODX 3.0.3, apache 2.4.54 Debian 11,  10.5.18-MariaDB-0+deb11u1, firefox

**Comment by halftrainedharry** [comment]
This should be fixed with #16302.

**Comment by intersel** [comment]
Ho great,  I did a search but missed this one!
